lib/ansible/plugins/test/same_file.yml

```diff
@@ -0,0 +1,24 @@
+DOCUMENTATION:
+  name: same_file
+  author: Ansible Core
+  version_added: "2.5"
+  short_description: compares 2 paths to see if they resolve to the same filesystem object
+  description: Check if the provided paths map to the same location onthe controller's filesystem (localhost)
+  aliases: [is_file]
+  options:
+    _input:
+      description: a path
+      type: path
+      required: true
+    _path2:
+      description: another path
+      type: path
+      required: true
+
+EXAMPLES: |
+  amionelevelfromroot: "{{ '/etc/hosts' is same_file('../etc/hosts') }}"
+
+RETURN:
+  _value:
+    description: Returns C(True) the paths correspond to the same location on the filesystem on the controller, c(False) if otherwise
```

Comment:
```suggestion
    description: Returns C(True) if the paths correspond to the same location on the filesystem on the controller, C(False) if otherwise.
```